### ISSUE TITLE
[Bug report] Error occurs when sending the same file from PC to android phone.

### ISSUE THREAD

**Issue by gannaiju** [localsend/localsend#1079]
### Describe the bug

Error [500] Could not save file occurs when sending the same file from PC to android phone.

### To reproduce

1. Send a file, say test.txt from PC to android phone.
2. After success, move test.txt to other directory.
3. Resend test.txt from PC to the android phone.
4. Error code " [500] Could not save file " occurs.

### Expected behavior

Sending the same file from PC to android phone successfully.

### Desktop (please complete the following information)

- OS: Windows 10 Home Edition
- Version: 1.13.0 (40)

### Smartphone (please complete the following information)

- Device: OnePlus 7 Pro
- OS: Hydrogen OS 11.0.9.1.GM21 (based on Android 11)
- Version: 1.13.1 (42)

**Comment by Caesarovich** [comment]
Hi @gannaiju , I could not reproduce your issue.

Is the issue still occurring on your side ? If so could you rephrase in details how you reproduced it.

**Comment by Alexprox7** [comment]
Having the same issue
